.couch_email_authrc-dist

```diff
@@ -0,0 +1,16 @@
+host = 127.0.0.1
+port = 3000
+couchDbBaseUrl = http://admin:secret@localhost:5984
+usersDb = _users
+sessionTimeout = 1
```

Comment:
For configuration settings where the meaning of the value is not obvious from the context, we should provide a short explanation in a comment.

For instance:

```ini
tokenExpiryTime = 1 ; the time in seconds the authentication token (which is part of the sign-in link) is valid 
```